.github/workflows/testrb.yml

```diff
@@ -31,38 +27,66 @@ jobs:
+  #           psql -d postgresql://postgres:postgres@postgres/postgres -c "CREATE DATABASE ${{ matrix.pg_database }} OWNER ${{ matrix.pg_role }};"
+  #           psql -d postgresql://${{ matrix.pg_role }}:${{ matrix.pg_password }}@postgres/${{ matrix.pg_database }} -c 'CREATE EXTENSION IF NOT EXISTS "pg_trgm"; CREATE EXTENSION IF NOT EXISTS "hstore"; CREATE EXTENSION IF NOT EXISTS "uuid-ossp";'
 
-            sudo chmod -R +x chemotion_ELN/spec
-            sudo chown -R gitlab-runner:gitlab-runner chemotion_ELN
+  #     - name: bundle + yarn install
+  #       working-directory: /home/gitlab-runner/chemotion_ELN
+  #       run: |
+  #           /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && bundle install"
+  #           /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && yarn install"
 
-      - name: bundle + yarn install
-        working-directory: /home/gitlab-runner/chemotion_ELN
-        run: |
-            /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && bundle install"
-            /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && yarn install"
+  #     - name: db
+  #       working-directory: /home/gitlab-runner/chemotion_ELN
+  #       run: |
+  #           /bin/bash -l -c "RAILS_ENV=test bundle exec bin/rails db:migrate db:test:prepare  > /dev/null"
+  #           /bin/bash -l -c "RAILS_ENV=test bundle exec bin/rails db:seed > /dev/null"
 
-      - name: db
-        working-directory: /home/gitlab-runner/chemotion_ELN
-        run: |
-            /bin/bash -l -c "RAILS_ENV=test bundle exec bin/rails db:migrate db:test:prepare  > /dev/null"
-            /bin/bash -l -c "RAILS_ENV=test bundle exec bin/rails db:seed > /dev/null"
+  #     - name: rb test
+  #       working-directory: /home/gitlab-runner/chemotion_ELN
+  #       run: |
+  #         /bin/bash -l -c "RAILS_ENV=test bundle exec rspec --exclude-pattern spec/{features}/**/*_spec.rb"
 
-      - name: rb test
-        working-directory: /home/gitlab-runner/chemotion_ELN
-        run: |
-          /bin/bash -l -c "RAILS_ENV=test bundle exec rspec --exclude-pattern spec/{features}/**/*_spec.rb"
+  #     - name: upload coverage
+  #       uses: actions/upload-artifact@v3
+  #       with:
+  #         name: coverage
+  #         path: /home/gitlab-runner/chemotion_ELN/coverage/lcov/chemotion_ELN.lcov
 
+  # coverage_report:
+  #   needs: docker_container
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #   - uses: actions/checkout/@v3  # we need to have the repository checked out in order to generate HTML report
+  #   - name: download coverage
+  #     uses: actions/download-artifact@v3
+  #     with:
+  #       name: coverage
+  #   - name: post coverage
+  #     uses: zgosalvez/github-actions-report-lcov@v1
+  #     with:
+  #       coverage-files: chemotion_ELN.lcov
+  #       minimum-coverage: 50
+  #       artifact-name: code-coverage-report
+  #       github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is this commented out? By mistake?

Reply:
I commented this out in order to test the checkout action in [924ae33](https://github.com/ComPlat/chemotion_ELN/pull/938/commits/924ae33b455c36cbd65b70fc9a475c9e44fca9c2). The checkout action is not related to the coverage. I'll convert to a draft until I get the checkout to work properly. The checkout currently fails because it's done manually w/o taking CI runs into account that are triggered by PRs:

https://github.com/ComPlat/chemotion_ELN/blob/5b9bdf04531c3d06a0d5aef3c5cb756f372a5f33/.github/workflows/testacceptance.yml#L44